### ISSUE TITLE
MSVC failed compile with -m32 after -m64

### ISSUE THREAD

**Issue by crackedmind** [vlang/v#9151]
**V version:** 
**OS:** 

**What did you do?**

**What did you expect to see?**

 
**What did you see instead?**

**Comment by revosw** [comment]
Which one of these behaviors should be correct?
- `-m32` takes precedence, ignore `-m64`
- `-m64` takes precedence, ignore `-m32`
- Output a specific error for when both `-m32` and `-m64` are defined at the same time (gets my vote, I usually prefer explicit behavior that clearly tells you what you need to know)

**Comment by JalonSolov** [comment]
Yes, it should be an error if both are defined at the same time, since they conflict with each other.
